tasks/install_yum.yml

```diff
@@ -29,9 +29,26 @@
     - "postgresql{{ postgresql_version_terse }}"
     - "postgresql{{ postgresql_version_terse }}-contrib"
 
-- name: PostgreSQL | PGTune
+- name: PostgreSQL | PGTune | yum
   yum:
     name: pgtune
     state: present
   environment: "{{ postgresql_env }}"
   when: postgresql_pgtune
+
+- name: PostgreSQL | Install pip | yum | RedHat v6
+  yum:
+    name: python-pip
+  when: ansible_distribution_major_version == '6' and
```

Comment:
I'm really perplexed as to why it's called `python-pip` on CentOS v6, but `python2-pip` on CentOS v7.  If there is a more generic way to fix this, then please let me know.

Reply:
I would create vars file for `RedHat-7.yaml` and `RedHat-6.yaml` and get package name as a variable with construction:
```
...
with_first_found:
 ...
```
Then it will be only one task here.

Reply:
:+1: it's what I do too in https://github.com/OCA/ansible-odoo/blob/master/tasks/main.yml#L3

Reply:
Thanks.  I have a PR #289 which adds multiple variables files for different versions... but it's stuck.  I guess I can merge some of those changes into here, and hope that doesn't create a merge conflict. :-O